factorio-plugin/global.lua

```diff
@@ -13,7 +13,7 @@ local function replace(uri, text, diffs)
       this_mod = "FallbackModName"
     else
       local workspace = require("workspace")
-      this_mod = (workspace.uri--[[@as string]] or workspace.getRootUri(uri)):match("[^/\\]+$")
+      this_mod = (workspace.rootUri or workspace.getRootUri(uri)):match("[^/\\]+$")
```

Comment:
`workspace.uri` is from `2.5.3` - it's intentional for backwards compatability. And btw this is actually broken, I fixed it in [this commit](https://github.com/JanSharp/FactorioSumnekoLuaPlugin/commit/d10b2c55832568447fda81a43c7eca1daa977251).